### ISSUE TITLE
🐛 Frontmatter blocks change embed behavior

### ISSUE THREAD

**Issue by matthewturk** [ebullient/obsidian-slides-extended#69]
### Input data (markdown source)

I'm trying to make some composable presentations.  If I do an embed, for instance:

```
![[Multiple Slides]]
```

in a presentation, it correctly embeds it if `Multiple Slides.md` does not include a frontmatter block.  For instance, if it is:

```
# Multiple Slides

Hi

---

Hello there!!

---

Slide three.
```

But, if it includes a frontmatter block, only the first slide is included.

### Expected Behavior

I'd like it to embed all, regardless.

### Current behaviour

Only the first one is embedded.

### Steps to reproduce

1. Create new presentation
2. Embed something with frontmatter
3. Embed something without

### Which Operating Systems are you using?

- [ ] Android
- [ ] iPhone/iPad
- [X] Linux
- [ ] macOS
- [ ] Windows

### Obsidian Version

1.7.7

### Slides Extended Plugin Version

2.1.3

### Checks

- [X] I have tried it in the sandbox vault with only this plugin enabled

### Possible solution

So this is not that important -- it's straightforward enough to work around it!  But I thought I'd report it just in case the fix was obvious.  (Maybe something in the regexes?)

**Comment by ebullient** [comment]
Fixed by 0ca4b19528cb6bae111777259bc6095f21ee1a3b

**Comment by matthewturk** [comment]
@ebullient Thank you so much!
